packages/box/tsconfig.json

```diff
@@ -19,11 +19,11 @@
     "typeRoots": [
       "./typings",
       "node_modules/@types",
-      "../../**/node_modules/@types/mocha",
```

Comment:
the ol' double star not cutting it, eh

Reply:
So those typing packages get installed at the root level and not in `packages/box/node_modules` because I guess they are used somewhere else in the repo as well.